src/Components/Links.js

```diff
@@ -28,41 +25,38 @@ function Links({ links }) {
         {links
           .filter((link) => Object.keys(colors).includes(link.icon))
           .map((link, index) => (
-            <Link
-              to={{ pathname: link.url }}
-              target="_blank"
+            <Button
               key={`link.url_${index}`}
+              onMouseOver={(e) => {
+                MouseOver(e, colors[link.icon])
+              }}
```

Comment:
Could this be 1 line for better readability?

```js
onMouseOver={(e) => MouseOver(e, colors[link.icon])}
```

Reply:
updated